### ISSUE TITLE
Training Results

### ISSUE THREAD

**Issue by guansidi** [Shaosifan/FIANet#6]
Hello, I am very interested in your work. Could you explain why, after following your steps to set up the environment and train the model, both the accuracy and IoU are 0? I look forward to your reply.

**Comment by Shaosifan** [comment]
> Hello, I am very interested in your work. Could you explain why, after following your steps to set up the environment and train the model, both the accuracy and IoU are 0? I look forward to your reply.

It is strange. Can you provide the training logs？

**Comment by guansidi** [comment]
Thank you for your reply. I made some improvements to your model by adjusting the learning rate and weight decay. In the first run, the model achieved meaningful evaluation metrics and the performance kept improving. However, when I run it again with exactly the same parameters, all the metrics drop to zero, even though the loss keeps decreasing. Could this be due to inappropriate hyperparameter settings? It's quite strange.


------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "Shaosifan/FIANet"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2025年5月16日(星期五) 中午11:38
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [Shaosifan/FIANet] Training Results (Issue #6)



 Shaosifan left a comment (Shaosifan/FIANet#6)
  
Hello, I am very interested in your work. Could you explain why, after following your steps to set up the environment and train the model, both the accuracy and IoU are 0? I look forward to your reply.
  
It is strange. Can you provide the training logs？

**Comment by Shaosifan** [comment]
> Thank you for your reply. I made some improvements to your model by adjusting the learning rate and weight decay. In the first run, the model achieved meaningful evaluation metrics and the performance kept improving. However, when I run it again with exactly the same parameters, all the metrics drop to zero, even though the loss keeps decreasing. Could this be due to inappropriate hyperparameter settings? It's quite strange.
> […](#)

This is indeed strange. I haven't encountered this situation before.
